Exec/hydro_tests/acoustic_pulse/convergence_ppm.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# echo the commands
+set -x
+
+DIM=3
+EXEC=./Castro${DIM}d.gnu.MPI.ex
+
+RUNPARAMS="castro.ppm_type=1 castro.time_integration_method=0"
+
+mpiexec -n 8 ${EXEC} inputs.3d.64 ${RUNPARAMS} amr.plot_file=acoustic_pulse_64_ppm_plt &> 64.out
```

Comment:
should it be inputs.64 instead of inputs.3d.64 , the same for the other input names on this file

Reply:
yes, you are right.  I renamed them after I wrote the script.  I'll fix it now.